glean-core/metrics.yaml

```diff
@@ -391,3 +391,41 @@ glean.upload:
     expires: never
     no_lint:
       - COMMON_PREFIX
+
+  pending_pings_directory_size:
+    type: memory_distribution
+    description: >
+      The size of the pending pings directory upon initialization of Glean.
+      This does not include the size of the deletion request pings directory.
+    memory_unit: kilobyte
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1601550
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1601550#c3
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - glean-team@mozilla.com
+    expires: never
+    no_lint:
+      - COMMON_PREFIX
+
+  deleted_pings_after_quota_hit:
```

Comment:
The idea is to know how many pings we have lost from going over quota. Yes, it could be incremented several times, but in the end it will still report how many pings were deleted because of going over quota, so, in my opinion, still good information.

In desktop we do have a histogram for this, I do agree that a distribution could also make sense here, specially in case we wanted to know specifically the average of pings deleted per scanning of the pings directory.

From my initial understanding of the usage for this metric, a counter works just fine. But now I am not so sure anymore. @Dexterp37 what do you think?

Reply:
I talked with @Dexterp37 a bit about this offline. He does agree it would be nice to have a distribution so that we can tell exactly how many pings were deleted per time we reach the quota, right now though we only have memory distrbutions or timing distribuitons and for this we would need a [custom distribution](https://mozilla.github.io/glean/book/user/metrics/custom_distribution.html), which is not available for usage outside of gecko view.

The counter at least will tell us how many pings we are losing from reaching quota, it's arguable how necessary that is. Let me know what you think.